### PR TITLE
OSDOCS-8517 Monitoring 4.15 RN bugs

### PR DESCRIPTION
**Pull request by cbippley**
Version(s): 4.15

Issue:[ [OSDOCS-8517](https://issues.redhat.com//browse/OSDOCS-8517)](https://issues.redhat.com/browse/OSDOCS-8517)

Link to docs preview: [Bug Fixes Section](https://71696--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-bug-fixes)

QE review:
Not needed

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -655,7 +655,7 @@ For more information, see xref:../monitoring/config-map-reference-for-the-cluste
 
 [id="ocp-4-15-monitoring-improved-alert-querying-for-user-defined-projects"]
 ==== Improved alert querying for user-defined projects
-Applications in user-defined projects now have API access to query alerts for application namespaces via the rules tenancy port for Thanos Querier. 
+Applications in user-defined projects now have API access to query alerts for application namespaces via the rules tenancy port for Thanos Querier.
```
Comment (ShaunaDiaz) [inline release_notes/ocp-4-15-release-notes.adoc:658]:
```suggestion
Applications in user-defined projects now have API access to query alerts for application namespaces by through the rules tenancy port for Thanos Querier.
```
https://www.ibm.com/docs/en/ibm-style?topic=word-usage#v

File: release_notes/ocp-4-15-release-notes.adoc [hunk 2]
```diff
@@ -1201,6 +1201,13 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-4-15-monitoring-bug-fixes"]
 ==== Monitoring
 
+* Previously, the monitoring-plugin component did not start if IPv6 was disabled for a cluster. This release updates the component to support the following internet protocol configurations in a cluster: IPv4 only, IPv6 only, and both IPv4 and IPv6 at the same time. This change resolves the issue, and the monitoring-plugin component now starts up if the cluster is configured to support only IPv6. (link:https://issues.redhat.com/browse/OCPBUGS-21610[*OCPBUGS-21610*])
```
Comment (ShaunaDiaz) [inline release_notes/ocp-4-15-release-notes.adoc:1204]:
```suggestion
* Previously, the monitoring-plugin component did not start if IPv6 was disabled for a cluster. This release updates the component to support the following internet protocol configurations in a cluster: IPv4 only, IPv6 only, and both IPv4 and IPv6 simultaneously. This change resolves the issue, and the monitoring-plugin component now starts up if the cluster is configured to support only IPv6. (link:https://issues.redhat.com/browse/OCPBUGS-21610[*OCPBUGS-21610*])
```

**Review by JoeAldinger — APPROVED** [review]
/lgtm, merging!

**Pull request merged**